core/src/main/java/org/apache/accumulo/fate/Fate.java

```diff
@@ -284,7 +287,7 @@ public void seedTransaction(long tid, Repo<T> repo, boolean autoCleanUp) {
 
         store.setProperty(tid, DEBUG_PROP, repo.getDescription());
 
-        store.setStatus(tid, TStatus.IN_PROGRESS);
+        store.setStatus(tid, TStatus.SUBMITTED);
```

Comment:
I think there are some OPs that do work in the `isReady()` function so I wonder if changing when it gets set to IN_PROGRESS will affect them. I will see if I can find one.

Reply:
The BulkImport and Compact operations do things in the `isReady()` function. I don't know if it matters if the status is IN_PROGRESS or SUBMITTED though.

Reply:
Also the `ShutdownTServer` operation